isamples_metadata/taxonomy/OpenContextClassifierInput.py

```diff
@@ -17,11 +17,16 @@ def build_text(self, description_map, labelType):
         concatenated_text = ""
         for key, value in description_map.items():
             value = str(value)
-            if labelType == "material" and key != self.OC_material_field:
+            if value == "":
+                continue
+            elif key == "context label":
```

Comment:
It would be helpful to add a comment here of what the original source value looks like so we have an idea of what the code is supposed to be operating on.